ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -2859,6 +2859,17 @@ public sealed class Bot : IAsyncDisposable {
 					SteamFriends.SetPersonaState(BotConfig.OnlineStatus);
 				}
 
+				if (BotConfig.VRMode){
+					ClientMsgProtobuf<global::SteamKit2.Internal.CMsgClientChangeStatus> requestVRMode = new(EMsg.ClientChangeStatus) {
```

Comment:
Extract this to standalone `ArchiHandler` function, we don't want low-level SK2 packets logic in `Bot`.